chainstate/launcher/src/lib.rs

```diff
@@ -19,25 +19,35 @@ mod config;
 
 use std::sync::Arc;
 
+use chainstate::InitializationError;
+use chainstate_storage::{BlockchainStorageRead, BlockchainStorageWrite};
+use storage_lmdb::resize_callback::MapResizeCallback;
+
 // Some useful reexports
 pub use chainstate::{
     chainstate_interface::ChainstateInterface, ChainstateConfig, ChainstateError as Error,
     DefaultTransactionVerificationStrategy,
 };
 pub use common::chain::ChainConfig;
 pub use config::{ChainstateLauncherConfig, StorageBackendConfig};
-use storage_lmdb::resize_callback::MapResizeCallback;
 
 /// Subdirectory under `datadir` where LMDB chainstate database is placed
 pub const SUBDIRECTORY_LMDB: &str = "chainstate-lmdb";
 
+pub const CHAINSTATE_STORAGE_VERSION_UNINITIALIZED: u32 = 0;
+pub const CHAINSTATE_STORAGE_VERSION_V1: u32 = 1;
+pub const CURRENT_CHAINSTATE_STORAGE_VERSION: u32 = CHAINSTATE_STORAGE_VERSION_V1;
```

Comment:
This could be a custom type wrapper over u32 rather than plain u32, or even an enum with repr u32. These constants could then be turned into associated constants like `StorageVersion::CURRENT` etc.

Reply:
ignoring for now